front/src/applications/operationalStudies/components/Scenario/ScenarioContent.tsx

```diff
@@ -212,11 +219,13 @@ const ScenarioContent = ({
                         : t('noElectricalProfileSet')}
                     </div>
                   </div>
-                  <MicroMacroSwitch isMacro={isMacro} setIsMacro={toggleMicroMacroButton} />
+                  {!collapsedTimetable && (
```

Comment:
This is included in an element which is only displayed when `collapsedTimetable` is truthy, so I think this is never hit?

Reply:
Apparently, we had 2 MicroMacroSwitch rendered, just deleted the second one. thanks